rpc/block.go

```diff
@@ -361,3 +361,43 @@ func nilToZero(f *felt.Felt) *felt.Felt {
 	}
 	return f
 }
+
+func (h *Handler) JunoGetBlockWithTxsAndReceipts(id BlockID) (*BlockWithTxs, *BlockWithReceipts, *jsonrpc.Error) {
```

Comment:
You will also need to add this as an option in 
func (h *Handler) Methods() ([]jsonrpc.Method, string) { //nolint: funlen

same for the V0_6 Methods

Reply:
This will expose the method for RPC calls (currently there is no way for users to make an rpc call to this method)

Reply:
resolved